tests/test_cargo_alias_config.rs

```diff
@@ -0,0 +1,51 @@
+use support::{project, execs, basic_bin_manifest};
+use hamcrest::{assert_that};
+
+fn setup() {}
+test!(default_alias_config_overrides_config {
+    let p = project("foo")
+        .file("Cargo.toml", &basic_bin_manifest("foo"))
+        .file("src/main.rs", r#"
+            fn main() {
+        }"#)
+        .file(".cargo/config","\
+            [alias]
+            b = \"not_build\"
```

Comment:
I'd recommend using raw strings (e.g. `r#" ... "#`) to avoid having to escape things here